src/libecalc/common/stream.py

```diff
@@ -15,9 +14,9 @@ class Config:
         alias_generator = to_camel_case
         allow_population_by_field_name = True
 
-    name: Optional[str]
-    rate: TimeSeriesStreamDayRate
-    pressure: TimeSeriesFloat
+    name: str
+    rate: Optional[TimeSeriesStreamDayRate]
+    pressure: Optional[TimeSeriesFloat]
     fluid_density: Optional[TimeSeriesFloat] = None
```

Comment:
At some point we need to put composition of fluid model in here, not just the density. Different streams entering the compressor train/system/network should be allowed to have different fluid models.